src/script/components/app-card.ts

```diff
@@ -27,15 +28,15 @@ export class AppCard extends LitElement {
   @property({ type: Boolean }) featured = false;
   @property({ type: Boolean }) shareLink = false;
   @property({ type: Array }) tags = [];
+  @property({ type: Boolean }) isActionCard = false;
 
   static get styles() {
     const defaultCardCss = css`
       fast-card {
         color: var(--font-color);
         font-size: var(--font-size);
         background: white;
-        min-width: 278px;
-        max-width: 416px;
+        width: max(278px, 416px);
```

Comment:
Maybe I'm missing part of the CSS max function, but I thought it was meant for max of different measure terms, e.g. `max(20vw, 400px)`. What does max do when handed two values of the same measure?

Reply:
You are right the first value is the default locked value and whenever the size of the screen grows it will go up to 416px max. Max always uses whichever value is bigger